src/components/CourseChecklist/CourseChecklist.test.jsx

```diff
@@ -425,5 +446,16 @@ describe('CourseChecklist', () => {
     it('getCommentSection returns null for unknown checklist item id', () => {
       expect(wrapper.instance().getCommentSection('test')).toBeNull();
     });
+
+    it('calls trackEvent when an update link is clicked', () => {
+      wrapper = shallowWithIntl(<CourseChecklist {...defaultProps} />);
+
+      const updateLink = wrapper.find(Hyperlink).at(0);
```

Comment:
Possibly. In this case I'm grabbing the first link simply because I want to test that an update link is calling the function at all. So I am trying to test that an update link that is generated calls to track the event - not that a _specific link_ itself calls the track event, which is why I chose to grab the first link that I found instead of adding an ID to get a specific button.

Reply:
Let's just leave it as is and deal with it if it breaks.